site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,461 @@
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+### Further reading
+
+Check out  [openthread.io](https://openthread.io/) and [GitHub](https://github.com/openthread) for a variety of OpenThread resources, including:
+
+*  [Supported Platforms](https://openthread.io/platforms/)
+    — discover all the platforms that support OpenThread
+    
+*  [Build OpenThread](../../guides/build/index.md)
+    — further details on building and configuring OpenThread
+    
+*  [Thread Primer](../../guides/thread-primer/index.md)
+    — covers all the Thread concepts featured in this Codelab
+    
+* [Silicon Labs OpenThread Training](https://www.silabs.com/support/training/introduction-openthread)
+
+    — an introduction to OpenThread, discusses commissioning and border routers, and a hands-on exercise creating an OpenThread network.
+
+* [QSG170: Silicon Labs OpenThread Quick-Start Guide](https://www.silabs.com/documents/public/quick-start-guides/qsg170-openthread-sdk-quick-start-guide.pdf)
+
+    — covers in detail the getting started process for Silicon Labs Thread development
+
+* [AN1256: Using the Silicon Labs RCP with the OpenThread Border Router](https://www.silabs.com/documents/public/application-notes/an1256-using-sl-rcp-with-openthread-border-router.pdf)
+
+    — Build and Installation Instructions for the Raspberry Pi Border Router Host
+
+## Optional firmware set up - Demos
+
+Duration: 3:00
+
+Demos are prebuilt firmware images that are ready to download to a compatible device. The quickest way to find if a demo is available for your part in Simplicity Studio is by clicking on your part under Debug Adapters view and then navigating to the EXAMPLE PROJECTS & DEMOS tab in the Launcher Perspective. Disable the Example Projects filter and check the Thread radio checkbox under Technology Type.
+
+![Studio demos](./img/studio_demos.png)
+
+Precompiled demo application images provided with the OpenThread SDK are compatible with the following boards: 
+
+1. BRD4161a
+
+2. BRD4166a
+
+3. BRD4168a
+
+4. BRD4180a
+
+5. BRD4304a
+
+   This list may be updated in the future SDK releases to include more radio boards. Please refer to the Silicon Labs OpenThread SDK release notes under Documentation for a full list of supported parts.
+
+Flash the following demos on the respective boards. To flash, select your board under Debug Adapters on the left and click on RUN for the corresponding sample applications. A pop up window will show the flash progress.
+
+1. BRD4166A: **ot-rcp**
+
+   This device will function as a Radio Co Processor to the OT Border Router. We will use this device to create a Thread network and onboard the other two devices on the Thread network. Being a Border Router this device also acts as a gateway for the devices in the Thread network to communicate over the internet.
```

Comment:
Can you have the text as part of the list item rather than separated?  So like

1. BRD4166A: **ot-rcp** — This device will...